Composer/packages/client/src/recoilModel/dispatchers/webchat.ts

```diff
@@ -4,14 +4,23 @@
 
 import { ConversationTrafficItem } from '@botframework-composer/types';
 import { useRecoilCallback, CallbackInterface } from 'recoil';
+import { v4 as uuid } from 'uuid';
 
 import { webChatTrafficState, webChatInspectionDataState, isWebChatPanelVisibleState } from '../atoms';
 import { WebChatInspectionData } from '../types';
 
+const addIdToTrafficItem = (item: ConversationTrafficItem) => {
+  if (!item.id) {
+    item.id = uuid();
```

Comment:
nit: are we opposed to just using the timestamp which is a number as the id?

Reply:
On occasion, I observed scenarios where several items had the same timestamp

Reply:
good call, why is it optional tho?

Reply:
On the server the traffic items are instantiated in one spot, and then stamped in the web socket server on their way to the client with whatever "traffic type" they have. I basically made it optional so I could just add it onto the object later without worrying about it at the time of instantiation.

These items are also instantiated in the client at certain points. So i could make it mandatory, but I would have to add the GUID generating code to each of these places. This seemed easier to manage.

Reply:
I'd call it something else, ```id``` is usually non optional 😄

Reply:
All the other potential property names I can think of that still preserve the meaning are basically just synonyms of `id`. So I'm just going to make the property mandatory and initialize to empty string on the server.